### PR TITLE
Fill Objects for MeVPrtl in StandardRecord

### PR DESCRIPTION
**Pull request by gputnam**
Title. Depends on https://github.com/SBNSoftware/sbnanaobj/pull/21 and https://github.com/SBNSoftware/sbncode/pull/141.

### CODE REVIEW

**Comment by gputnam** [comment]
In the ordering, https://github.com/SBNSoftware/sbncode/pull/141 should be merged in first and then we can point this one to develop.

**Comment by gputnam** [comment]
Since https://github.com/SBNSoftware/sbncode/pull/141 is now merged I edited this to point to develop.

**Review by mroda88 — APPROVED** [review]
LGTM

File: sbncode/CAFMaker/FillTrue.cxx [hunk 1]
```diff
@@ -109,6 +109,42 @@ namespace caf {
...
+   srtruth.ray_weight = truth.ray_weight;
+   srtruth.decay_weight = truth.decay_weight;
+   srtruth.C1 = truth.C1;
+   srtruth.C2 = truth.C2;
+   srtruth.C3 = truth.C3;
+   srtruth.C4 = truth.C4;
+   srtruth.C5 = truth.C5;
+
+   switch(truth.gen) {
+     case evgen::ldm::kDissonantHiggs:
```
Comment (mroda88) [inline sbncode/CAFMaker/FillTrue.cxx:137]: We need to change this following the changes in sbnanaobj.

**Review by mroda88 — CHANGES_REQUESTED** [review]
I think now there is a problem of compatibility with https://github.com/SBNSoftware/sbnanaobj/pull/21#

File: sbncode/CAFMaker/FillTrue.cxx (hunk 1 above)
Comment (gputnam) [reply sbncode/CAFMaker/FillTrue.cxx:137]: No actually the way I changed the code there's no change to sbncode. There is now an update to sbnana, which is in another PR.

**Review by mroda88 — APPROVED** [review]
Coo, we are good to go

**Pull request merged**